CHANGELOG.yaml

```
@@ -11,6 +11,14 @@ meta:
     - clients: the service API changed in a way that requires clients to be updated
 
 changes:
+  - title: Emulate HTTP redirects and retry-after in DSS file proxy and endpoint change to `fetch/dss/files/{uuid}`
```

```suggestion
  - title: Emulate HTTP redirects and retry-after in DSS file proxy
```

You original version isn't even correct English.